science/2022-11-23-science-blog-6/index.md

```diff
@@ -1,15 +1,261 @@
+
+### 2.2 PCR
+
+In total, there were approximately 9.8 million reported PCR test results (PCR), of which approximately 2.7 million were preceded by at least one risk exposure. The following figure shows the progression of positivity rates over time for each type of risk notification. PCR test results have been registered via data donation since 10 March 2021.
+
+<figure class="figure-img">
+<img src="./ppa_risk_infection_ct_timeline_pcr-1_en.png" class="figure-img img-fluid" alt="Figure 2: Positivity rate after risk notification over time (PCR)."/>
+<figcaption aria-hidden="true">Figure 2: Positivity rate after risk notification over time (PCR).</figcaption>
+</figure>
+
+What is striking here is the significantly greater positivity rate for increased risk compared to that for lower risk without a risk exposure in times of relatively lower incidence. In times of high incidence, the curves run almost the same. Below we will look at this relationship in more detail.
+
+The positivity rate also varies here depending on the infection event and warning. After a red warning, the positivity rate is between 0 and 0.88, for a lower risk with risk exposure between 0 and 0.8 and for a low risk without risk exposure between 0 and 0.78.
+
+The [RKI‑test figures](https://www.rki.de/DE/Content/InfAZ/N/Neuartiges_Coronavirus/Daten/Testzahlen-gesamt.html) are given for comparison. The somewhat greater proportion of persons tested positive in the CWA data donation is due to the different underlying survey population. For example, people under the age of 16 are not formally allowed to use the Corona‑Warn‑App and are therefore under-represented in its user base. On the other hand, the RKI figures also increasingly include routine and pooling test results, which are probably rarely, if ever registered via the Corona‑Warn‑App and, in accordance with their purpose, are more often negative.
+
+A comparison with the positivity rate from the [event-related user survey (EDUS)](../2021-08-02-science-blog-3) from the early year 2021 also shows a parallel trend (at a lower level) to the positive share of red warned persons. The difference can be explained as follows: Healthy people and those with milder courses of illness took part in the survey. Those who were very ill probably had other concerns than participating in this survey. For example, on 22/04/2021, about 19% (287/(287+1218)) of CWA data donors had a positive PCR test result at increased risk. For EDUS, it was only about 15% (20/(20+110)). With 6 additional positives, it would also have been 19% here. This suggests that about a quarter of those red warned and positive tested persons had a non-mild course. One could also hypothesise that EDUS participants are more conscientious – after all, they participated in the voluntary survey – and therefore implement more measures to avoid infection.
```

Comment:
"22/04/2021"
don't mix date formats, use
"22 April 2021"

Reply:
will use April 22, 2021, same format as above and all the webpage.

Reply:
@dsarkar 

That makes sense, after all the work that went in to standardizing the other pages regarding dates to use Month day, year in English!